crates/bevy_ecs/src/query/state.rs

```diff
@@ -334,6 +396,10 @@ where
         );
     }
 
+    /// Runs `f` on each query result in parallel using the given task pool.
```

Comment:
```suggestion
    /// Runs `f` on each query result in parallel using the given `task_pool`.
```

This change also applies for all other `par_` functions